docs/application/dotnet/guides/nui/videoview.md

```diff
@@ -1,58 +1,99 @@
+The following basic example shows how to create a `VideoView` object:
 
-In this tutorial, the following subjects are covered:
+**Figure: Basic VideoView**
 
-[VideoView event](#1)<br>
-[Creating a VideoView](#2)<br>
-[VideoView Properties](#3)<br>
+![Basic VideoView](./media/basicVideoview.png)
 
-<a name="1"></a>
-## VideoView event
+1.  To use the `VideoView` class, add the following namespaces:
 
-The following table lists the basic signal provided by the `VideoView` class.
+    ```cs
+    using Tizen.NUI;
+    using Tizen.NUI.BaseComponents;
+    ```	    
+  
+2.   The video file is assumed to be in the resources directory. Create an instance of the `VideoView` class and use the `ResourceUrl` property to pass the path to the video file. Set `WidthResizePolicy` to make `VideoView` instance use full width of the window and set `HeightResizePolicy` to maintain aspect ratio of video. In the end invoke `Play()` method to start video:
 
-**Table: VideoView input signals**
+      ```cs
+      VideoView player = new VideoView();
+      player.ResourceUrl = DirectoryInfo.Resource + "sample.mp4";
+      player.WidthResizePolicy = ResizePolicyType.FillToParent;
+      player.HeightResizePolicy = ResizePolicyType.DimensionDependency;
+      player.Play();
 
-| Input signal  | Description                                 |
-| ------------- | ------------------------------------------- |
-| `Finished`    | Emitted when a video playback is finished.  |
+      Window.Instance.Add(player);
+      ```
 
-The `VideoView` class provides the `Finished`, which is emitted when the video playback is finished. The related callback can support some basic actions.
+> **Note**  
+> You can set the video file to be played in the `new VideoView()` function, or by modifying `Video` property.
 
-<a name="2"></a>
-## Creating a VideoView
 
-The following basic example shows how to create a `VideoView` object:
+## VideoView Methods
 
-```
-// Create a ScrollView instance
-Window window = Window.Instance;
-VideoView view = new VideoView( "videofile.mp4" );
-window.Add(view);
-view.ParentOrigin = ParentOrigin.Center;
-view.PivotPoint = PivotPoint.Center;
-view.HeightResizePolicy = ResizePolicyType.UseNaturalSize;
-view.WidthResizePolicy = ResizePolicyType.UseNaturalSize;
-view.Size2D = new Size2D( WIDTH, HEIGHT );
-view.Play();
-```
+  As it was mentioned before, the `VideoView` provides minimum functions for playback:
 
- > **Note**  
- > You can set the video file to be played in the `new VideoView()` function, or by modifying `Video` property.
+  1. The `Play()` method starts video:
+
+      ```cs
+      player.Play();
+      ```
+
+  2. The `Pause()` method pauses video. Video can be resumed by using the `Play()` method:
+
+      ```cs
+      player.Pause();
+      ```
+
+  2. The `Stop()` method stops video. After the `Play()` method is called, the video will be started from the beginning:
```

Comment:
3. The `Stop()` method stops video. To play the video, use the `Play()` method, the video will play from the  beginning: